contacts24/models/address_book.py

```diff
@@ -24,19 +24,19 @@ def delete(self, name: str) -> None:
             print(f"No records found by the name {name}")
 
     def save_to_file(self, filename: str = ADDRESSBOOK_FILE) -> None:
-        with open(filename, "w") as f:
-            records_list = [record.to_dict() for record in self.data.values()]
-            json.dump(records_list, f)
```

Comment:
@krisdzhus  as we are not using `json` directly here, please remove the import for it.
It's not critical, but it's best practice